SUMMARY.md

```diff
@@ -150,18 +150,18 @@
     * [WalkerMove](api-native-functions/walkermove.md)
     * [WalkerTurn](api-native-functions/walkerturn.md)
     * [WalkerDiff](api-native-functions/walkerdiff.md)
-    * Resolve_Buddies
-    * Opposite_Step
+    * Resolve\_Buddies
```

Comment:
Underscores are the same. 
Anybody cloning this repo rather than the book may try to use the backsplash as part of the command (it was slightly confusing when I copied them out from the pdf manual!)

Reply:
Same as with parenthesis. Old outdated stuff.

Reply:
I am not sure how to solve it, maybe some filter at the end. When I type in GitBookEditor it automatically add escape chars...